src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.cs

```diff
@@ -23,8 +23,8 @@ public void M(int x, int y)
 }
 ";
             string expectedOperationTree = @"
-IOperation:  (OperationKind.None) (Syntax: '(x, x + y)')
-  Children(2): IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'x')
+ITupleExpression (OperationKind.TupleExpression, Type: (System.Int32 x, System.Int32)) (Syntax: '(x, x + y)')
+  Elements(2): IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'x')
```

Comment:
Filed [https://github.com/dotnet/roslyn/issues/19866](https://github.com/dotnet/roslyn/issues/19866)

---
In reply to: [118046117](https://github.com/dotnet/roslyn/pull/19636#discussion_r118046117) [](ancestors = 118046117)